packages/cosmic-proto/proto/stride/icaoracle/tx.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+package stride.icaoracle;
+
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+import "amino/amino.proto";
+
+option go_package = "github.com/Stride-Labs/stride/v25/x/icaoracle/types";
+
+// Msg defines the Msg service.
+service Msg {
+  // Adds a new oracle given a provided connection
+  rpc AddOracle(MsgAddOracle) returns (MsgAddOracleResponse);
+  // Instantiates an Oracle CW contract
+  rpc InstantiateOracle(MsgInstantiateOracle)
+      returns (MsgInstantiateOracleResponse);
+  // Restores the oracle ICA channel after a closure
+  rpc RestoreOracleICA(MsgRestoreOracleICA)
+      returns (MsgRestoreOracleICAResponse);
+  // Toggle's whether an oracle is active and should receive metric updates
+  rpc ToggleOracle(MsgToggleOracle) returns (MsgToggleOracleResponse);
+  // Removes an oracle completely
+  rpc RemoveOracle(MsgRemoveOracle) returns (MsgRemoveOracleResponse);
+}
+
+// Adds a new oracle
+message MsgAddOracle {
```

Comment:
I think these are the first app/chain-specific protos we're adding to this package. There seems to be a lot of non-user facing stuff in here.  There's probably limited harm to including everything, but making sure it's a cognizant decision to do so.

A quick sweep it seems these are essential:
- `stakeibc`, `staketia`, `stakedym` and their dependencies `records`, `epochs`

These also seem to be user facing, but not needed for the `.liquidStake` story:
- `airdrop`, `autopilot`, `claim`, `vesting`

If anything, maybe just remove `icaoracle`, `interchainquery`, `mint` (if it doesn't affect codegen via import deps)

Reply:
Only what ends up in the package exports will be used and I had only added `stride/stakeibc/tx`. But better not to include files that aren't exported. I'll keep it to staking packages (and deps) but I'll wait for @99adarsh to chime in before I merge.

Reply:
@turadg @0xpatrickdev  keeping only stakeibc, staketia, stakedym and their dep makes sense.